src/backend/distributed/citus.sql

```diff
@@ -103,6 +103,27 @@ SET search_path = 'pg_catalog';
 
 /* master_* functions */
 
+CREATE FUNCTION citus_table_size(relation_name text)
+    RETURNS integer
```

Comment:
I think it is possible that return value is bigger than integer. Maybe we can use bigint type instead?